packages/amplify-graphql-schema-generator/src/datasource-adapter/mysql-datasource-adapter.ts

```diff
@@ -116,10 +134,21 @@ export class MySQLDataSourceAdapter extends DataSourceAdapter {
   }
 
   private async loadAllFields(): Promise<void> {
-    this.fields = [];
     // Query INFORMATION_SCHEMA.COLUMNS table and load fields of all the tables from the database
-    const columnResult = (await this.dbBuilder.raw(`SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = '${this.config.database}'`))[0];
-    this.fields = columnResult.map((item: any) => {
+    const LOAD_FIELDS_QUERY = `SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = '${this.config.database}'`;
+    this.fields = [];
+    let columnResult;
+    if (this.useVPC && this.vpcSchemaInspectorLambda) {
+      columnResult = await invokeSchemaInspectorLambda(this.vpcSchemaInspectorLambda, this.config, LOAD_FIELDS_QUERY, this.vpcLambdaRegion);
+    }
+    else {
+      columnResult = (await this.dbBuilder.raw(LOAD_FIELDS_QUERY))[0];
+    }
```

Comment:
Prefer to avoid `let` if possible. This can just be a ternary.

Reply:
Updated as suggested.